atomic_reactor/plugins/post_tag_from_config.py

```diff
@@ -43,7 +44,38 @@ class TagFromConfigPlugin(PostBuildPlugin):
 
     TAGS_FILENAME = 'additional-tags'
 
-    def get_tags(self):
+    def __init__(self, tasker, workflow, tags=None):
+        super(TagFromConfigPlugin, self).__init__(tasker, workflow)
+        self.tags = tags
+
+    def parse_and_add_tags(self):
+        class MyFormatter(string.Formatter):
```

Comment:
Can we do a bit of refactoring, and instead of copying this class, move it to a common place? Maybe utils.py, or a new formatters.py. Let's also call it something useful, like `LabelFormater`.